web/src/components/Layout.tsx

```diff
@@ -94,7 +94,7 @@ const Main = styled.main<{ fullWidth: boolean }>`
 `
 
 const Aside = styled.aside<{ languageSelectorHeight: number }>`
-  top: ${props => props.languageSelectorHeight + dimensions.headerHeightLarge}px;
+  top: ${props => props.languageSelectorHeight + dimensions.headerHeightLarge + dimensions.additionalWidth}px;
```

Comment:
Nicely done, working around the magic-number limitation! I'm not sure the constant should be called `additionalWidth` though since it isn't additional space in the horizontal but in the vertical direction. How about `additionalSpacing` instead?